common/src/main/java/com/alibaba/nacos/common/remote/client/grpc/GrpcClient.java

```diff
@@ -103,22 +111,17 @@ public GrpcClient(String name) {
     
     public GrpcClient(String name, Properties configProperties) {
         super(name);
-        initGrpcClient(configProperties);
+        initGrpcClient(configProperties, NACOS_CLIENT_GRPC_THREADPOOL_KEEPALIVETIME, NACOS_CLIENT_GRPC_TIMEOUT,
```

Comment:
Why list constants? If you want make them as an list or array, you can create an constant list in class or `Constants`